$OpenBSD: patch-configure,v 1.6 2018/09/07 13:25:02 naddy Exp $

Skip libc crypto tests.  If these succeed, the libcrypto tests for
additional functions will never be run.

Index: configure
--- configure.orig
+++ configure
@@ -19096,7 +19096,7 @@ fi
 
 
 case "$host_os" in
-  *mingw* | *cygwin* | *msys*)
+  *mingw* | *cygwin* | *msys* | openbsd*)
 	;;
   *)
 
